lib/brakeman/report/report_codeclimate.rb

```diff
@@ -0,0 +1,70 @@
+require "json"
+require "yaml"
+
+class Brakeman::Report::CodeClimate < Brakeman::Report::Base
+  DOCUMENTATION_PATH = File.expand_path("../../../../docs/warning_types", __FILE__)
+  REMEDIATION_POINTS_CONFIG_PATH = File.expand_path("../config/remediation.yml", __FILE__)
+  REMEDIATION_POINTS_DEFAULT = 300_000
+
+  def generate_report
+    all_warnings.map { |warning| issue_json(warning) }.join("\0")
+  end
+
+  private
+
+  def issue_json(warning)
+    check = warning.check.sub("Brakeman::Check", "")
+    path = warning.file.sub("#{tracker.app_path}/", "")
+    body = content_for(warning.warning_type, warning.link)
+
+    {
+      type: "Issue",
+      check_name: check,
```

Comment:
Unless I'm misunderstanding how you will map it. Otherwise I'd recommend going with the `warning_type` which is more like a category.